cli.go

```diff
@@ -205,7 +208,7 @@ func uninstallGlobal() {
 
 // Check latest version of git-hooks by github release
 // If there are new version of git-hooks, download and replace the current one
-func update(releases []github.RepositoryRelease) {
+func update(releases []*github.RepositoryRelease) {
```

Comment:
thanks for optimize this, I'm definitely just a Golang amateur and this is my first Golang project 😄